api/api.yaml

```diff
@@ -1672,6 +1672,75 @@ paths:
             application/problem+json:
               schema:
                 $ref: "../problem/problem.yaml#/components/schemas/ApiProblem"
+  /projects/{projectIdOrName}/connections:create:
+    post:
+      summary: Create a new connection (only valid for certain providers)
+      operationId: createConnection
+      tags: ["Connection"]
+      parameters:
+        - name: projectIdOrName
+          in: path
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - projectId
+                - groupRef
+                - consumerRef
+                - provider
+              properties:
```

Comment:
there aren't any info about the "secrets" here (e.g. API Key, Client ID, Client Secret, etc.)

Reply:
it's not necessary, at least for the auth type I'm adding ("none" -- which I need for the mock connector which I need both for testing as well as the jitter PR).

I can circle back and add those once I need them.